bin/lesson_initialize.py

```diff
@@ -119,41 +119,37 @@
 
 ## Using GitHub
 
-If you choose to contribute via GitHub,
-you may want to look at
+If you choose to contribute via GitHub, you may want to look at
 [How to Contribute to an Open Source Project on GitHub][how-contribute].
-In brief:
-
-1.  The published copy of the lesson is in the `gh-pages` branch of the repository
-    (so that GitHub will regenerate it automatically).
-    Please create all branches from that,
-    and merge the [master repository][repo]'s `gh-pages` branch into your `gh-pages` branch
-    before starting work.
-    Please do *not* work directly in your `gh-pages` branch,
-    since that will make it difficult for you to work on other contributions.
-
-2.  We use [GitHub flow][github-flow] to manage changes:
-    1.  Create a new branch in your desktop copy of this repository for each significant change.
-    2.  Commit the change in that branch.
-    3.  Push that branch to your fork of this repository on GitHub.
-    4.  Submit a pull request from that branch to the [master repository][repo].
-    5.  If you receive feedback,
-        make changes on your desktop and push to your branch on GitHub:
-        the pull request will update automatically.
-
-Each lesson has two maintainers who review issues and pull requests
-or encourage others to do so.
-The maintainers are community volunteers,
-and have final say over what gets merged into the lesson.
+A published copy of the lesson is available in the `gh-pages` branch of the master repository.
+Before starting work, please make sure your clone of the master `gh-pages` branch is up-to-date
+and create your own revision-specific branch(es) from there.
+Please only work on your newly-created branch(es) and *not*
+your clone of the master `gh-pages` branch.
+To manage changes, we follow [GitHub flow][github-flow]. In brief:
+
+1.  Fork the repository to your GitHub profile and create a clone on your desktop.
```

Comment:
This section should be about contribute via GitHub (web interface). Why have a local copy?

Reply:
I can change the wording to make it done through the web interface instead.  I included the desktop version because I was maintaining the content of what was previously in the contributing.rmd file while making it clearer.  It should be fairly straightforward to change.